.travis.yml

```diff
@@ -2,22 +2,11 @@
 language: ruby
 
 rvm:
-  - 1.9.3
-  - 2.1.0
+  - 2.1.9
   - 2.3.1
 
 env:
   matrix:
-    - PUPPET_GEM_VERSION="~> 3.1.0" CHECK=test
-    - PUPPET_GEM_VERSION="~> 3.2.0" CHECK=test
-    - PUPPET_GEM_VERSION="~> 3.3.0" CHECK=test
-    - PUPPET_GEM_VERSION="~> 3.4.0" CHECK=test
-    - PUPPET_GEM_VERSION="~> 3.5.0" CHECK=test
-    - PUPPET_GEM_VERSION="~> 3.6.0" CHECK=test
-    - PUPPET_GEM_VERSION="~> 3.7.0" CHECK=test
-    - PUPPET_GEM_VERSION="~> 3.8.0" CHECK=test
-    - PUPPET_GEM_VERSION="~> 3.8.0" CHECK=build DEPLOY_TO_FORGE=yes
-    - PUPPET_GEM_VERSION="~> 3" FUTURE_PARSER="yes" CHECK=test
     - PUPPET_GEM_VERSION="~> 4.0.0" CHECK=test
```

Comment:
Do we want to explicitly support all releases of v4? I'm thinking of only supporting the current release and the two before it. Things really do change in minor versions and you'd be surprised how it can add it. It also greatly increases testing time.